openreview/conference/builder.py

```diff
@@ -1678,7 +1678,7 @@ def get_attachment_file(submission):
         for future in futures:
             result = future.result()
 
-    def post_decision_stage(self, reveal_all_authors=False, reveal_authors_accepted=False, decision_heading_map=None, submission_readers=None):
+    def post_decision_stage(self, reveal_all_authors=False, reveal_authors_accepted=False, decision_heading_map=None, submission_readers=None, hide_fields=[]):
```

Comment:
instead of passing `hide_fields` as a parameter, can you add this field to the SubmissionStage class and get the value from there?

I would like to do the same for `setup_post_submission_stage`